dask/array/tests/test_array_core.py

```diff
@@ -3647,3 +3647,21 @@ def func(a, b):
 
     assert_eq(da.map_blocks(func, x, y, chunks=(2,), dtype=x.dtype),
               np.array([99, 9, 199, 19, 299, 29, 399, 39]))
+
+
+def test_nbytes_auto():
+    chunks = normalize_chunks("800B", shape=(500,), dtype='float64')
+    assert chunks == ((100, 100, 100, 100, 100),)
+    chunks = normalize_chunks("200B", shape=(10,10), dtype='float64')
+    assert chunks == ((5, 5), (5, 5))
+    chunks = normalize_chunks("33B", shape=(10,10), dtype='float64')
+    assert chunks == ((2, 2, 2, 2, 2), (2, 2, 2, 2, 2))
+    chunks = normalize_chunks("1800B", shape=(10, 20, 30), dtype='float64')
+    assert chunks == ((5, 5), (5, 5, 5, 5), (6, 6, 6, 6, 6))
```

Comment:
Thoughts on also adding a test that includes a tuple-valued chunks value like `normalize_chunks((10, "100 B"), ...)`